tests/test_analog_signals.py

```diff
@@ -64,6 +83,11 @@ def test_acromag_readback(fake_acromag):
     assert fake_acromag.ai1_13.get() == 1.0
 
 
+def test_acromag_ch_readback(fake_acromag_in, fake_acromag_out):
+    assert fake_acromag_in.channel.get() == 0.23
+    assert fake_acromag_out.channel.get() == 0.33
```

Comment:
Probably also good to try a `.put()` on the output channel and test that it changed.

Reply:
Added one in 1381498  - thank you